packages/react/src/hooks/useAuth.ts

```diff
@@ -141,6 +142,10 @@ export const useAuth: UseAuth = () => {
     [orgId, orgRole, userId, orgPermissions],
   );
 
+  useEffect(() => {
```

Comment:
We need to be careful that those events will be emitted on the React tree render. Especially during local development, this could be dispatched in a huge volume due to hot reloads.

I'm considering passing a sampling rate to decrease the fraction of all events that should be recorded. We already do so for `COMPONENT_MOUNTED`.